frontend/src/views/LoginView.vue

```diff
@@ -1,13 +1,51 @@
 <template>
 	<div class="page_box_wrapper">
-		<div class="page_box">
-			<h1>This is a login page</h1>
-		</div>
+        <h1> HI WANNA PLAY PONG? </h1>
+
+            <div class="c_block c_form_group">
+                <InputField label="Username" placeholder="username" v-model="loginForm.name"/>
+                <InputField label="Password" placeholder="password" v-model="loginForm.password"/>
+
+                <div class="c_block c_split">
+                    <p>
+                        <a href="#" v-on:click.prevent="userStore.login(loginForm)">Login</a>
+                        /
+                        <a href="/register">Register</a>
+                    </p>
+                    <p>
+                        <a :href="redirectUrl">Sign in with 42</a>
+                    </p>
+                </div>
+            </div>
```

Comment:
Sooo, here I was wrong
Login with the <a> element is annoying (no action on enter)

this is my suggestion:

```vue
<form
	method="post"
	action=""
	class="c_block c_form_group"
	@submit.prevent="userStore.login(loginForm)"
>
<!-- inputfields -->

<!-- instead of this:
<a href="#" v-on:click.prevent="userStore.login(loginForm)">Login</a> -->
<input class="link_button" type="submit" value="Login" />
</form>
```

and this added to main.css somewhere
```css
.link_button {
	-webkit-appearance: none;
	appearance: none;
	background: none;
	border: 0px;
	padding: 0;
}
```